pkgs/development/node-packages/composition-v10.nix

```diff
@@ -1,4 +1,4 @@
-# This file has been generated by node2nix 1.7.0. Do not edit!
+# This file has been generated by node2nix 1.6.0. Do not edit!
```

Comment:
do you mind updating your node2nix version and trying again :)

Reply:
Done! (Issue was that `nixpkgs-19.03-darwin` doesn't have the latest version.)